### PR TITLE
Add Remark Command - Chandan

### LINKED ISSUE #84: Implement Remark Command

### PR DESCRIPTION
**Pull request by Chandan8186**
Fix #84 
From "Tutorial 2: Adding a new Command"

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/EditCommand.java [hunk 1]
```diff
@@ -192,6 +196,14 @@ public Optional<Address> getAddress() {
             return Optional.ofNullable(address);
         }
 
+        public void setRemark(Remark remark) {
+            this.remark = remark;
+        }
+
+        public Optional<Remark> getRemark() {
+            return Optional.ofNullable(remark);
```
Comment (timleow) [inline src/main/java/seedu/address/logic/commands/EditCommand.java:204]: Great use of `Optional` here

**Review by timleow — APPROVED** [review]
LGTM!

**Comment by Chandan8186** [comment]
Refactoring of person to employee has been merged with my "Remark" implementation and conflicts resolved. There should be no occurrences of "person" in any form in the code files.

File: src/main/java/seedu/address/model/util/SampleDataUtil.java [hunk 2]
```diff
@@ -11,31 +11,35 @@
...
 import seedu.address.model.employee.Phone;
+import seedu.address.model.employee.Remark;
 import seedu.address.model.tag.Tag;
 
 /**
  * Contains utility methods for populating {@code AddressBook} with sample data.
  */
 public class SampleDataUtil {
+
+    public static final Remark EMPTY_REMARK = new Remark("");
```
Comment (antonTan96) [inline src/main/java/seedu/address/model/util/SampleDataUtil.java:22]: nice use of DRY

**Review by antonTan96 — APPROVED** [review]
LGTM!

File: src/main/java/seedu/address/ui/EmployeeCard.java [hunk 3]
```diff
@@ -51,6 +53,7 @@ public EmployeeCard(Employee employee, int displayedIndex) {
         name.setText(employee.getName().fullName);
         phone.setText(employee.getPhone().value);
         address.setText(employee.getAddress().value);
+        remark.setText(employee.getRemark().value);
```
Comment (antonTan96) [inline src/main/java/seedu/address/ui/EmployeeCard.java:56]: Maybe we could individually test this function but it's probably optional.

Comment (Chandan8186) [reply src/main/java/seedu/address/ui/EmployeeCard.java:56]: Yup we can consider that

**Pull request merged**